packages/backend/migrations/20240820101148_drop_unique_grants.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.alterTable('grants', function (table) {
+    table.dropUnique(['authServerId', 'accessType', 'accessActions'])
```

Comment:
Dropping the unique constraint on authServerId accessType and accessActions such that we can properly soft delete grants (see migration below)

Reply:
seems similar to this case? https://github.com/interledger/rafiki/pull/2863#discussion_r1720065804

So could we use a partial unique index (where `deletedAt` is not null)?

Reply:
I don't think this unique constraint is very useful to begin with, particularly because order matters in `accessActions` array, meaning [Create, Read] is different from [Read, Create]